doc/releasenotes/16_0_0_summary.md

```diff
@@ -3,27 +3,27 @@
 ### Table of Contents
 
 - **[Major Changes](#major-changes)**
-  - **[VReplication](#vreplication)**
-    - [VStream Copy Resume](#vstream-copy-resume)
-    - [VDiff2 GA](#vdiff2-ga)
-  - **[Tablet throttler](#tablet-throttler)**
-  - **[Incremental backup and point in time recovery](#inc-backup)**
-  - **[Replication manager removal and VTOrc becomes mandatory](#repl-manager-removal)**
   - **[Breaking Changes](#breaking-changes)**
     - [VTGate Advertised MySQL Version](#advertised-mysql-version)
     - [Default MySQL version on Docker](#default-mysql-version)
     - [Running Vitess on the Operator](#running-vitess-on-the-operator)
+    - [`lock-timeout` and `remote_operation_timeout` Changes](#lock-timeout-introduction)
+    - [Orchestrator Integration Deletion](#orc-integration-removal)
     - [vtctld UI Removal](#vtcltd-ui-removal)
     - [vtctld Flag Deprecation & Deletions](#vtctld-flag-deprecations)
-    - [Orchestrator Integration Deletion](#orc-integration-removal)
     - [mysqlctl Flags](#mysqlctl-flags)
```

Comment:
I think we should have an unified subsection for flag deletion. As of now, we have several subsections:
- `vtctld Flag Deprecation & Deletions`
- `mysqlctl Flags`
- `Removed flag`
- vtorc's `Flag Deprecations`

If we had only 1 sections `Flag Deletions and Deprecations` under `Breaking Changes` and then have sub category per binary, it would be easier for the reader.